server/lib/routes/get-index.js

```diff
@@ -9,9 +24,40 @@ module.exports = function () {
   route.path = '/';
 
   route.process = function (req, res) {
-    res.render('index');
+    if (! shouldEmitActivityEvent()) {
+      return res.render('index');
+    }
+
+    var time = Date.now();
+    var flowId = crypto.randomBytes(16).toString('hex');
+    res.render('index', { flowBeginTime: time, flowId: flowId });
+
+    setImmediate(function () {
+      // This event data is structured to be consistent with the auth server
+      // activity events. So yes, the format is different to the content server
+      // log lines. And yes, there is a mix of camelCase and under_score keys.
+      var activityEvent = {
+        device_id: null, //eslint-disable-line camelcase
+        event: 'flow.begin',
+        flow_id: flowId, //eslint-disable-line camelcase
+        flow_time: 0, //eslint-disable-line camelcase
+        new_user: false, //eslint-disable-line camelcase
```

Comment:
I suggest omitting this field because we'll *never* know whether or not it's a new-user flow at this point in time, we can fill in a default in the backend.